content/docs/user-guide/scripting/script-canvas/programmer-guide/behavior-context.md

```diff
@@ -1,10 +1,10 @@
 ---
-description: ' Learn about the relationship between Script Canvas and the behavior
-  context in Open 3D Engine. '
-title: 'Script Canvas and the Behavior Context'
+title: Script Canvas and the Behavior Context
+description: Learn about the important relationship between Script Canvas and the behavior context in Open 3D Engine.
+weight: 100
 ---
 
-You can use Script Canvas to expose runtime code in a visual authoring environment using the behavior context. In addition to this topic, it is recommended that you also read the [Programmer's Guide to Entities and Components](/docs/user-guide/components/development/) and [Behavior Context](/docs/user-guide/components/development/behavior-context/).
+You can use Script Canvas to expose runtime code in a visual authoring environment using the behavior context. In addition to this topic, it is recommended that you also read the [Programmer's Guide to Component Development](/docs/user-guide/components/development/) and [Behavior Context](/docs/user-guide/components/development/behavior-context).
```

Comment:
We're recommending the other resources because they're good resources on the generic concept of "Behavior Context" in O3DE. This page covers behavior context in the context of Script Canvas. Perhaps I should explain that here?